Game/lv/Game.po

```diff
@@ -1005,7 +1005,7 @@ msgstr "Runā Džonatans Derils. Šis ieraksts ir paredzēts jebkuram, kurš to
 #: /Game/Resources/Localisation/ST_Recordings.ST_Recordings
 msgctxt "Recordings,JohnDiary1Part10"
 msgid "I've given up hope of ever holding my daughter again or looking into my wife's eyes just one more time."
-msgstr "Esmu atmetis cerību, ka jebkad vēlreiz varēšu apskaut savu meitu un skatīties savas sievas acīs."
+msgstr "Esmu atmetis cerību, ka kādreiz atkal varēšu samīlēt savu meitu un skatīties savas sievas acīs."
```

Comment:
I do not believe samīlēt is the correct verb - from some cursory research this seems to mean romantic love. The original sentence implies platonic love.

Reply:
@alanedwardes It's a word often used in casual conversation to describe hugging. But ok, I changed it.

Reply:
Understood - if it's a better fit then feel free to restore. Reviewing these is always quite hard :)